From 11c3032bf4f39bad3cebc348f6c8337aea916fd3 Mon Sep 17 00:00:00 2001
From: Yuri Tolstov <yuri.tolstov@cavium.com>
Date: Thu, 18 Oct 2018 10:46:57 +0300
Subject: [PATCH 0320/1051] octeontx-pko: Reduce number of FPA buffers used by
 DQs

New setting (4096 buffers) will give:
Memory usage = 4096 * 4096 (buffer size) = 16,777,216 bytes.
Packets per PKO = ((4096 - 6) * 127 * 1022) / (127 + 1022) = 462,016

Signed-off-by: Yuri Tolstov <yuri.tolstov@cavium.com>
Signed-off-by: Yury Norov <ynorov@marvell.com>
Signed-off-by: Yury Norov <ynorov@caviumnetworks.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/cavium/octeontx-83xx/pkopf_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/cavium/octeontx-83xx/pkopf_main.c b/drivers/net/ethernet/cavium/octeontx-83xx/pkopf_main.c
index f93b2b490dde..191a99c1bf80 100644
--- a/drivers/net/ethernet/cavium/octeontx-83xx/pkopf_main.c
+++ b/drivers/net/ethernet/cavium/octeontx-83xx/pkopf_main.c
@@ -1099,7 +1099,7 @@ static int setup_dpfi(struct pkopf *pko)
 		symbol_put(fpavf_com);
 		return -ENODEV;
 	}
-	buffers = 80000;
+	buffers = 4096;
 
 	err = fpavf->setup(fpa, buffers, pko->pdm_buf_size,
 			FPA_VF_FLAG_CONT_MEM);
-- 
2.17.1

